README.md

```diff
@@ -1,4 +1,13 @@
-# greenstar
+# GreenSTAR
+
+[![Build Status](https://github.com/arikkfir/greenstar/actions/workflows/build.yaml/badge.svg)](https://github.com/arikkfir/greenstar/actions/workflows/)
+[![Go Report Card](https://goreportcard.com/badge/github.com/arikkfir/greenstar/backend)](https://goreportcard.com/report/github.com/arikkfir/greenstar/backend)
+![TypeScript](https://img.shields.io/badge/TypeScript-4.x-blue.svg)
```

Comment:
I think this badge is just an icon?

Reply:
Should i just remove the typescript badge then

Reply:
I'm ok with that for now, sure.

Reply:
Alright, I am removing it now

Reply:
Resolved changes see commit: 3e94a2c